build/schema.js

```diff
@@ -170,7 +170,7 @@ module.exports = {
       description: 'uuid',
       type: 'string',
       allowEmpty: false,
-      pattern: /^[0-9a-f]{62}$/
+      pattern: /^[0-9a-f]{16,62}$/
```

Comment:
All now done and fixed. There were some changes unbuilt from master too, which is weird.

I've added the CS headers too to make the diffs nicer. If you can approve if you're happy that'd be great.